File: JavaToKotlin/app/src/main/java/io/dustin/apps/board/domain/qna/answer/repository/AnswerRepository.java
package io.dustin.apps.board.domain.qna.answer.repository;

import io.dustin.apps.board.domain.qna.answer.model.Answer;
import io.dustin.apps.common.repository.BaseRepository;

public interface AnswerRepository extends BaseRepository<Answer, Long> {

    Answer findByQuestionId(Long questionId);
}

File: apps/src/main/kotlin/io/dustin/apps/board/domain/qna/answer/repository/AnswerRepository.kt
package io.dustin.apps.board.domain.qna.answer.repository

import io.dustin.apps.board.domain.qna.answer.model.Answer
import io.dustin.apps.common.repository.BaseRepository

interface AnswerRepository : BaseRepository<Answer, Long> {
    fun findByQuestionId(questionId: Long): Answer?
}
